### ISSUE TITLE
Inline toolbar buttons have no visual or programatic labels 

### ISSUE THREAD

**Issue by JeffersonBledsoe** [plone/volto#3318]
<img width="335" alt="Screenshot of the current Volto inline editing toolbar." src="https://user-images.githubusercontent.com/30210785/166742124-ede043bf-9d9d-4016-ab07-d30acde345c1.png">

**Describe the bug**
The current text editing toolbar relies solely on icons to describe what action they will perform. While some icons such as bold and italics are fairly universal, other icons such as the 'megaphone' icon are fairly ambiguous. Ontop of this, the icons/ buttons don't provide any kind of text description of the button, rendering all buttons ambiguous to assistive technology users and being a WCAG/ ATAG failure. 

## Solutions
- Text below icons:
  - Probably the 'best' user experience, but there isn't much room on the toolbar to provide this.
- Add a visually hidden label 
  - Required for assistive technology, such as screen readers, to be able to read the contents of the button
- Add a tooltip so visual users can understand 
  - Allows visual users who are using a mouse to understand the action of the button before performing it.
  - Not very helpful to touch users as they can't hover.
  - Not very helpful to speech-input users as they won't be able to visually see the accessible name of the button. As the icons don't contain text, this doesn't fail WCAG 2.5.3.
  - This is done by the Gutenberg editor so some users may be familiar with this pattern.

### Gutenberg Toolbar
<img width="387" alt="Screenshot of the Wordpress Gutenberg editor's toolbar. The bold button has been hovered over so is highlighted blue and a tooltip displaying the word Bold with a keyboard shortcut of CMD + B is shown." src="https://user-images.githubusercontent.com/30210785/166745417-b705cec0-09b3-4f6e-b2aa-5c09f655aff3.png">

- Browser: All
- Volto Version:  15.8.0 and below
- Plone Version: N/A
- Plone REST API Version: N/A

**Comment by tisto** [comment]
+1. Though, since we will move to Slate editor soon it is not worth the effort to fix this for DraftJS.

**Comment by tiberiuichim** [comment]
With the Slate integration, we have the tooltips, see screenshot.

![image](https://user-images.githubusercontent.com/141568/166871558-f1c57177-c970-441d-b1b6-c249cbf97df6.png)

**Comment by davisagli** [comment]
Closing since this is solved in slate.

I do like how Gutenberg shows the related keyboard shortcut though
